include/ast.hpp

```diff
@@ -210,160 +211,30 @@ struct IntConstExprNode : public ExprNode {
 
 /// @note This is an abstract class.
```

Comment:
`UnaryExprNode` is now a concrete class.

Reply:
Nice catch!